movie_flow_data.sql

```diff
@@ -0,0 +1,153 @@
+VALUES
+    ('Leonardo', 'DiCaprio', 'American', 'Male', '1974-11-11'),
+    ('Song', 'Kang-ho', 'South Korean', 'Male', '1967-01-17'),
+    ('Jamie', 'Foxx', 'American', 'Male', '1967-12-13'),
+    ('Sally', 'Hawkins', 'British', 'Female', '1976-04-27'),
+    ('Charlize', 'Theron', 'South African', 'Female', '1975-08-07'),
+    ('Lady', 'Gaga', 'American', 'Female', '1986-03-28'),
+    ('Daniel', 'Craig', 'British', 'Male', '1968-03-02'),
+    ('Ginnifer', 'Goodwin', 'American', 'Female', '1978-05-22'),
+    ('Matthew', 'McConaughey', 'American', 'Male', '1969-11-04'),
+    ('Sandra', 'Bullock', 'American', 'Female', '1964-07-26'),
+	('Tina', 'Fey', 'American', 'Female', '1970-05-18'),
+    ('Graham', 'Norton', 'Irish', 'Male', '1963-04-04'),
+	('Charlize', 'Theron', 'South African', 'Female', '1975-08-07'),
+	('Lady', 'Gaga', 'American', 'Female', '1986-03-28'),
+    ('Bradley', 'Cooper', 'American', 'Male', '1975-01-05'),
+	('Ana', 'de Armas', 'Cuban', 'Female', '1988-04-30'),
+    ('Chris', 'Evans', 'American', 'Male', '1981-06-13'),
+	('Ginnifer', 'Goodwin', 'American', 'Female', '1978-05-22'),
+    ('Jason', 'Bateman', 'American', 'Male', '1969-01-14'),
+	('Matthew', 'McConaughey', 'American', 'Male', '1969-11-04'),
+    ('Anne', 'Hathaway', 'American', 'Female', '1982-11-12'),
+	('Sandra', 'Bullock', 'American', 'Female', '1964-07-26'),
+    ('Melissa', 'McCarthy', 'American', 'Female', '1970-08-26');
+	
+INSERT INTO def_external_platform (name, founded_date)
+VALUES 
+    ('Netflix', '1997-08-29'),
+    ('Hulu', '2007-10-29'),
+    ('Amazon Prime Video', '2006-09-07'),
+    ('Disney+', '2019-11-12'),
+    ('HBO Max', '2020-05-27');
+
+INSERT INTO def_provider_company (name, founded_date)
+VALUES 
+    ('DNEG', '1988-01-01'),
+    ('Barunson E&A', '1996-12-27'),
+    ('Pixar Animation Studios', '1986-02-03'),
+    ('Searchlight Pictures', '1994-01-01'),
+    ('Double Dare You', '1986-10-06'),
+    ('TSG Entertainment', '2012-01-01'),
+    ('Village Roadshow Pictures', '1954-01-01'),
+    ('Warner Bros', '1923-04-04'),
+    ('Live Nation Productions', '1995-01-25'),
+    ('Lions Gate Films', '1997-07-10'),
+    ('Walt Disney Animation Studios', '1923-10-16'),
+    ('Paramount Pictures', '1912-05-08'),
+    ('Legendary Entertainment', '2000-01-01');
+
+INSERT INTO rating (rating_source, rate, max_rate, movie_id)
+VALUES 
+    ('IMDb', 8.7, 10, 1),  -- Inception
+    ('Rotten Tomatoes', 9.0, 10, 2),  -- Parasite
+    ('Metacritic', 8.5, 10, 3),  -- Soul
+    ('IMDb', 7.9, 10, 4),  -- The Shape of Water
+    ('Rotten Tomatoes', 8.8, 10, 5),  -- Mad Max: Fury Road
+    ('IMDb', 8.2, 10, 6),  -- A Star Is Born
+    ('Rotten Tomatoes', 9.5, 10, 7),  -- Knives Out
+    ('Metacritic', 8.4, 10, 8),  -- Zootopia
+    ('IMDb', 8.6, 10, 9),  -- Interstellar
+    ('Rotten Tomatoes', 7.7, 10, 10);  -- The Heat
+
+INSERT INTO n2n_movie_to_actor (movie_id, actor_id)
+VALUES 
+    (1, 1),  -- Inception - Leonardo DiCaprio
```

Comment:
Nice to see that you commented it, this is a good practice while working with IDs 👍